com.gluonhq.SceneBuilder.yml

```diff
@@ -0,0 +1,55 @@
+app-id: com.gluonhq.SceneBuilder
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: SceneBuilder-runner
+rename-desktop-file: scenebuilder-SceneBuilder.desktop
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --persist=.scenebuilder
+  - --filesystem=home
+build-options:
+  arch:
+    x86_64:
```

Comment:
this has strictly no effect.

Sicen  you copied #3402, there is a hint over there.